react-native-onesignal.podspec

```diff
@@ -22,5 +22,5 @@ Pod::Spec.new do |s|
   # pod 'React', :path => '../node_modules/react-native/'
 
   # The Native OneSignal-iOS-SDK from cocoapods.
-  s.dependency 'OneSignal', '2.15.2'
+  s.dependency 'OneSignal', '>= 3.0.0-beta2', '< 4.0'
```

Comment:
Let's use specifically `3.0.0-beta2` so we know if someone is using exactly `react-native-onesignal-4.0.0-beta1` they have this.